### DISCUSSION TITLE
Win11: Can't install replaygain plugin - missing gobject-introspection-1.0?

### DISCUSSION THREAD

**Discussion by robuck86** [beetbox/beets#5515] [Q&A]
I've been trying to follow the instructions for installing the replaygain plugin, but I keep getting an error. I see I'm missing gobject-introspection-1.0 but I don't know how to install it. Can anyone help me get past this error?

PS C:\beets> pip install "beets[replaygain]"
Requirement already satisfied: beets[replaygain] in c:\users\rob86\appdata\local\programs\python\python313\lib\site-packages (2.1.0)
Collecting PyGObject (from beets[replaygain])
  Using cached pygobject-3.50.0.tar.gz (1.1 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Installing backend dependencies ... done
  Preparing metadata (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Preparing metadata (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [26 lines of output]
      + meson setup C:\Users\rob86\AppData\Local\Temp\pip-install-ovgt4wuw\pygobject_16be857e1bfc4fc8a6208544d278a07e C:\Users\rob86\AppData\Local\Temp\pip-install-ovgt4wuw\pygobject_16be857e1bfc4fc8a6208544d278a07e\.mesonpy-vsapix7g -Dbuildtype=release -Db_ndebug=if-release -Db_vscrt=md -Dtests=false -Dwheel=true --wrap-mode=nofallback --native-file=C:\Users\rob86\AppData\Local\Temp\pip-install-ovgt4wuw\pygobject_16be857e1bfc4fc8a6208544d278a07e\.mesonpy-vsapix7g\meson-python-native-file.ini
      The Meson build system
      Version: 1.6.0
      Source dir: C:\Users\rob86\AppData\Local\Temp\pip-install-ovgt4wuw\pygobject_16be857e1bfc4fc8a6208544d278a07e
      Build dir: C:\Users\rob86\AppData\Local\Temp\pip-install-ovgt4wuw\pygobject_16be857e1bfc4fc8a6208544d278a07e\.mesonpy-vsapix7g
      Build type: native build
      Project name: pygobject
      Project version: 3.50.0
      Activating VS 17.12.1
      C compiler for the host machine: cl (msvc 19.42.34433 "Microsoft (R) C/C++ Optimizing Compiler Version 19.42.34433 for x64")
      C linker for the host machine: link link 14.42.34433.0
      Host machine cpu family: x86_64
      Host machine cpu: x86_64
      Program python3 found: YES (C:\Users\rob86\AppData\Local\Programs\Python\Python313\python.exe)
      Run-time dependency python found: YES 3.13
      Did not find pkg-config by name 'pkg-config'
      Found pkg-config: NO
      Did not find CMake 'cmake'
      Found CMake: NO
      Run-time dependency gobject-introspection-1.0 found: NO (tried pkgconfig and cmake)
      Not looking for a fallback subproject for the dependency gobject-introspection-1.0 because:
      Use of fallback dependencies is disabled.

      ..\meson.build:31:9: ERROR: Dependency 'gobject-introspection-1.0' is required but not found.

**Comment by RollingStar** [comment]
Shot in the dark, it might be trying and failing to compile it. I'd try other way(s) to install pygobject as well as check your C++ redistributables (try to install more of them).

https://pygobject.gnome.org/getting_started.html

> Dependency 'gobject-introspection-1.0'

Sounds like you need this package somehow.
